packages/wdio-repl/src/index.js

```diff
@@ -118,7 +119,8 @@ export default class WDIORepl {
 
             this.isCommandRunning = false
             clearTimeout(timeout)
-            const commandError = new Error(e.message)
+            const errorMessage = e ? e.message : 'Command execution timed out'
```

Comment:
While debugging this test I encountered `e = undefined` which in turn threw error when `e.message` was attempted to be accessed.